bin/pycbc_live

```diff
@@ -412,6 +419,11 @@ class LiveEventManager(object):
                        'Followup detectors: {1}')
             comment = comment.format(ifo, ppdets(followup_ifos))
 
+            # Has a coinc event at this time been uploaded recently?
+            # If so, skip
+            if event.merger_time in self.last_ten_coinc_events_uploaded:
+                continue
```

Comment:
We probably want some tolerance on the merger time, maybe 100 ms? I would also `logging.info()` the fact that we are skipping an upload.